code/modules/organs/organ_internal.dm

```diff
@@ -190,15 +190,29 @@
 	if(owner.life_tick % PROCESS_ACCURACY == 0)
 
 		//High toxins levels are dangerous
-		if(owner.getToxLoss() >= 60 && !owner.reagents.has_reagent("anti_toxin"))
+		if(owner.getToxLoss() >= 30 && !owner.reagents.has_reagent("anti_toxin"))
 			//Healthy liver suffers on its own
 			if (src.damage < min_broken_damage)
-				src.damage += 0.2 * PROCESS_ACCURACY
+				src.damage += 0.3 * PROCESS_ACCURACY
 			//Damaged one shares the fun
 			else
 				var/obj/item/organ/internal/O = pick(owner.internal_organs)
 				if(O)
 					O.damage += 0.2  * PROCESS_ACCURACY
+		//So much !!FUN!!, a broken liver won't be able to handle smaller amounts of toxins either
+		if(owner.getToxLoss() >= 10 && !owner.reagents.has_reagent("anti_toxin"))
+			if (src.damage > min_broken_damage)
+				var/obj/item/organ/internal/O = pick(owner.internal_organs)
+				if(O)
+					O.damage += 0.2 * PROCESS_ACCURACY
```

Comment:
This block for damaging the liver will get called in addition to the above one if both the requirements are met, which doesn't seem intended. It's also unnecessary copy-paste of checks that will under circumstances (toxloss 30+) be called twice.
There's surely a nicer solution to this.